buildpipeline/Dotnet-CoreClr-Trusted-BuildTests.json

```diff
@@ -85,6 +85,27 @@
         "failOnStandardError": "false"
       }
     },
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": true,
```

Comment:
I don't think this one needs to be `alwaysRun` is `true`.  `AlwaysRun` says to run this step even if there was a previous failure and the build is going to fail.  I don't think we want to run this step in that scenario.